.eslintrc.js

```diff
@@ -162,14 +161,13 @@ const strictModeRules = {
 };
 
 const variablesRules = {
-	'init-declarations': TRANSITION_WARNING,
```

Comment:
This floods us with warnings and doesn't always make sense. There are a number of places we declare a variable and then if/else to determine its value.